src/main/java/com/dev/cinema/validation/FieldsValueMatchValidator.java

```diff
@@ -0,0 +1,27 @@
+package com.dev.cinema.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class FieldsValueMatchValidator implements ConstraintValidator<FieldsValueMatch, Object> {
+    private String filed;
+    private String fieldMatch;
+
+    public void initialize(FieldsValueMatch constraint) {
+        this.filed = constraint.field();
+        this.fieldMatch = constraint.fieldMatch();
+    }
+
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext constraintValidatorContext) {
+        Object fieldValue = new BeanWrapperImpl(value).getPropertyValue(filed);
+        Object fieldMatchValue = new BeanWrapperImpl(value).getPropertyValue(fieldMatch);
+
+        if (fieldValue != null) {
+            return fieldValue.equals(fieldMatchValue);
+        } else {
```

Comment:
user Objects.equals() instead of if else check for null

Reply:
done